jfr-streaming/build.gradle.kts

```
@@ -0,0 +1,62 @@
+val gsonVersion: String by project
+val slf4jVersion: String by project
+val otelVersion: String by project
+val otelAlphaVersion: String by project
+
+
+plugins {
+    id("com.github.johnrengelman.shadow") version "5.2.0"
```

```suggestion
    id("com.github.johnrengelman.shadow")
```

Version already defined in settings.gradle